src/main/java/com/rakovets/course/java/core/practice/operators/Task06.java

```diff
@@ -30,6 +30,14 @@ public static void main(String[] args) {
     static String getPlayingTime(int playingTimeInSeconds) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return null;
+        var D = 0;
+        var H = 0;
```

Comment:
"Violation: 'nameVariable' was not used for variable naming
Solution: Rename variable"